mesheryctl/Makefile

```diff
@@ -1,4 +1,4 @@
-OUTDIR := _output
+OUTDIR := bin
 
 ARCH := amd64
```

Comment:
Can you also reintroduce the command `make` here which was removed in commit https://github.com/layer5io/meshery/commit/f4d503c9628c11a806787b5c30c0aff8c0ba963a

Reply:
Oh that's my bad. Good catch. Will add it back